ted/src/main/java/frc/robot/subsystems/TofSubsystem.java

```diff
@@ -14,7 +14,6 @@
 
 import edu.wpi.first.math.util.Units;
 import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
-import edu.wpi.first.wpilibj2.command.CommandScheduler;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 
 
```

Comment:
So I think we should move this code into another sensor class instead of it being a subsystem.  Presumably we get the periodic for free (being a motivation as to why it was created as a subsystem), but I'd say its better to have the subsystems that need this sensor ask it for its state when they need it.

I will push a change I suggest ...

Reply:
I don't follow why this shouldn't be a subsystem. Are you worried about checking the sensor each tick rather than having the sensor read only when needed? We could keep it a subsystem, but just not have the read sensor in periodic. Instead, we could push the read into the two methods that access currentRangeInches. I think having readSensor() in periodic was very useful for debugging, but agree in the real robot it spams the can bus.

Reply:
@josieammer - I pushed a change that I'd suggest in this case.

Reply:
Looks good.